.gitignore

```diff
@@ -6,3 +6,4 @@ lib
 coverage
 react.js
 react-native.js
+.idea
```

Comment:
For non-project specific things like this, you might consider a global gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore